### ISSUE TITLE
Return schedule as array of objects / arrays

### ISSUE THREAD

**Issue by turtle0x1** [crunzphp/crunz#28]
**Description**  
Its cool that you can run `vendor/bin/crunz s:l` but it would be really good if we could see these jobs displayed on a web interface.

Unfortunately I'm not quite sure how to approach this issue so im requesting it but cant produce a PR (which I know is rude sorry).

Happy to give it a shot but will need some pointers :smile:

**Comment by PabloKowalczyk** [comment]
Will JSON output be ok?

**Comment by turtle0x1** [comment]
JSON would work better if not possible to call pragmatically withoutout opening a shell process. 

Ideally what Im trying todo is something like;

```
$crunz = new Crunz();
$schedules = $crunz->getSchedule();
// [ Task(), Task(), Task() ]
```

**Comment by PabloKowalczyk** [comment]
> JSON would work better if not possible to call pragmatically withoutout opening a shell process.

Currently it won't be easy, because there is no place where developer can hook after collecting all tasks.
Will `--format` option on `schedule:list` help?

**Comment by turtle0x1** [comment]
Yeah because you can do some hacky stuff like

```
$results = json_decode(shell_exec("vendor/bin/crunz s:l --format json"))
```

it might be worth thinking about the case where an exception / error happens as well and what will be out put on that case! 

If the errors aren't JSON you'd have todo something like;

```
$results = json_decode(shell_exec("vendor/bin/crunz s:l --format json"))
// Might not have access to JSON_THROW_EXCEPTION
if(json_last_error()){
     // If when crunz errors the exception isn't JSON we wont be able to parse a real error message
     throw new \Exception("Something went wrong in the crunz script")
}
```

**Comment by PabloKowalczyk** [comment]
Ok, give me few days :)

> it might be worth thinking about the case where an exception / error happens as well and what will be out put on that case!

In that case you should rely on process exit code or use `symfony/process` and `mustRun` (https://symfony.com/doc/current/components/process.html).

**Comment by PabloKowalczyk** [comment]
Ok, check PR https://github.com/crunzphp/crunz/pull/31.

**Comment by turtle0x1** [comment]
Awesome, thank you :smile: Will try to give this a test over the next couple of days!

**Comment by PabloKowalczyk** [comment]
Fixed by #31.
